central/localscanner/certificates.go

```diff
@@ -16,27 +18,45 @@ func IssueLocalScannerCerts(namespace string, clusterID string) (*storage.TypedS
 	if namespace == "" {
 		return nil, errors.New("namespace is required to issue the certificates for the local scanner")
 	}
-
+	var certs = map[storage.ServiceType]*storage.TypedServiceCertificate{
+		storage.ServiceType_SCANNER_SERVICE:    nil,
+		storage.ServiceType_SCANNER_DB_SERVICE: nil,
+	}
 	var certIssueError error
-	caPem, scannerCertificate, err := localScannerCertificatesFor(storage.ServiceType_SCANNER_SERVICE, namespace, clusterID)
-	if err != nil {
-		certIssueError = multierror.Append(certIssueError, err)
+	var caPem []byte
+
+	if features.ScannerV4Support.Enabled() && features.ScannerV4.Enabled() {
+		certs[storage.ServiceType_SCANNER_V4_INDEXER_SERVICE] = nil
+		certs[storage.ServiceType_SCANNER_V4_MATCHER_SERVICE] = nil
```

Comment:
 should Central issue cert of the Matcher service If Matcher is not going to be part of secured cluster?

Reply:
oh right good catch